types/aliases.ts

```diff
@@ -1,5 +1,7 @@
+import type * as cache from "@actions/cache";
 import type * as docker from "../src/docker.js";
 import type * as util from "../src/util.js";
 
+export type CacheType = typeof cache;
```

Comment:
Nit: I suppose this alias should be named `Cache` for consistency with the existing ones if that doesn't create any name conflicts.